Bigtable/src/BigtableClient.php

```diff
@@ -0,0 +1,193 @@
+            [
+                'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.UpdateInstanceMetadata',
+                'callable' => function ($instance) {
+                    $name = InstanceAdminClient::parseName($instance['name'])['instance'];
+                    return $this->instance($name, $instance);
+                }
+            ], [
+                'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.CreateInstanceMetadata',
+                'callable' => function ($instance) {
+                    $name = InstanceAdminClient::parseName($instance['name'])['instance'];
+                    return $this->instance($name, $instance);
+                }
+            ]
+        ]);
+    }
+
+    /**
+     * Lazily instantiate an instance.
+     *
+     * Example:
+     * ```
+     * $instance = $bigtable->instance('my-instance');
+     * ```
+     *
+     * @param string $instanceId The instance ID
+     * @param array $instance [optional] {
+     *     Configuration options
+     *
+     *     @type string $displayName **Defaults to** the value of $instanceId.
+     *     @type int $instanceType Possible values are represented by the following constants:
+     *           `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION` and
+     *           `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT`.
+     *           **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *     @type array $labels as key/value pair ['foo' => 'bar']. For more information, see
+     *           [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     * }
+     * @return Instance
+     */
+    public function instance($instanceId, array $instance = [])
+    {
+        return new Instance(
+            $this->connection,
+            $this->lroConnection,
+            $this->lroCallables,
+            $this->projectId,
+            $instanceId,
+            $instance
+        );
+    }
+
+    /**
+     * Constructs an array that contains cluster metadata.
+     *
+     * Example:
+     * ```
+     * $cluster = $bigtable->clusterMetadata('my-cluster', 'us-east-b');
+     * ```
+     *
+     * @param string $locationId The location ID
+     * @param int $storageType The storage media type for persisting Bigtable data. Possible values include
+     *            `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_SSD`,
+     *            `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_HDD` and
+     *            `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_UNSPECIFIED`.
+     *            **Defaults to** `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_UNSPECIFIED`.
+     * @param int $serveNodes
+     *
+     * @return array
+     */
+    public function clusterMetadata($clusterId, $locationId, $storageType = null, $serveNodes = null)
+    {
+        if ($clusterId == '') {
```

Comment:
I'd suggest using empty() as it covers the case of null values too.
At the very least, I'd use "===" instead of "==" to scope down what we want to verify here.